.github/workflows/markdown-link-check-config.json

```
@@ -23,6 +23,9 @@
         },
         {
             "pattern": "osquery.io/schema"
+        },
+        {
+            "patter": "github.com/issues"
```

```suggestion
            "pattern": "github.com/issues"
```
Or remove this since maybe it's not doing anything due to the typo?

---

Thanks @zwass 👓  Going to remove this and see if the Markdown link check gets tripped up.

---

Ok, it looks like the markdown link check failed after removal. Adding `github.com/issues` back with correct spelling of `pattern`